pai-management/doc/cluster-bootup.md

```diff
@@ -62,15 +87,17 @@ ssh-password: pai-password
 #service-cluster-ip-range: <ip-range-for-k8s>
 
 ```
+#### How to check
+check all configruation items of the quick-start.yaml are correct.
 
-### Step 2. Generate OpenPAI configuration files
+### Step 2. Generate OpenPAI configuration files <a name="c-step-2"></a>
 
 After the quick-start.yaml is ready, use it to generate four configuration yaml files as follows.
 
 ```
 python paictl.py cluster generate-configuration -i ~/quick-start.yaml -o ~/pai-config -f
 ```
-
+#### How to check
 The command will generate the following four yaml files.
```

Comment:
those files under the directory `-o ~/pai-config`

Reply:
ok